packages/http-client-java/Setup.ps1

```diff
@@ -0,0 +1,15 @@
+param (
+    [switch] $RebuildJar = $false
+)
+
+if ($RebuildJar) {
+    mvn clean install -f generator/pom.xml -DskipTests
+    if ($LASTEXITCODE -ne 0) {
+        exit $LASTEXITCODE
+    }
+}
+
+# re-build http-client-java
+npm install 
+npm run build:generator && npm run build:emitter && npm pack
```

Comment:
is pack needed here? Feels like there is maybe some process simplification that could happen